### PR TITLE
Keymap piv3rt

### PR DESCRIPTION
**Pull request by piv3rt**
## Description
This is piv3rt's keymap

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/dztech/dz60rgb/keymaps/piv3rt/keymap.c [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+        if (record->event.pressed) {
+            SEND_STRING(SS_DOWN(X_LALT) SS_TAP(X_P1) SS_TAP(X_P7) SS_TAP(X_P5) SS_UP(X_LALT));
+        }
+        return false;
+        break;
+    }
+    return true;
+};
+
+void rgb_matrix_indicators_user(void) {
```
Comment (drashna) [inline keyboards/dztech/dz60rgb/keymaps/piv3rt/keymap.c:161]: I would recommend using `rgb_matrix_indicators_advanced_user` function and the `RGB_MATRIX_INDICATOR_SET_COLOR` function in that, instead. 

https://docs.qmk.fm/#/feature_rgb_matrix?id=indicators

Comment (piv3rt) [reply keyboards/dztech/dz60rgb/keymaps/piv3rt/keymap.c]: Thanks for your input, I changed the function to the advanced one and made use of the macro.

File: keyboards/dztech/dz60rgb/keymaps/piv3rt/keymap.c [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+        return false;
+        break;
+    }
+    return true;
+};
+
+void rgb_matrix_indicators_advanced_user(uint8_t led_min, uint8_t led_max) {
+    led_t led_state = host_keyboard_led_state();
+
+    if (!g_suspend_state && rgb_matrix_config.enable) {
```
Comment (drashna) [inline keyboards/dztech/dz60rgb/keymaps/piv3rt/keymap.c:163]: This isn't needed anymore. Indicator functions only run if the LEDs are enabled.

```suggestion
```

File: keyboards/dztech/dz60rgb/keymaps/piv3rt/keymap.c [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+                RGB_MATRIX_INDICATOR_SET_COLOR(31 + i, 0x33, 0x66, 0x99);
+                RGB_MATRIX_INDICATOR_SET_COLOR(46 + i, 0x33, 0x66, 0x99);
+            }
+            RGB_MATRIX_INDICATOR_SET_COLOR(38, 0x33, 0x66, 0x99);
+            RGB_MATRIX_INDICATOR_SET_COLOR(39, 0x33, 0x66, 0x99);
+            RGB_MATRIX_INDICATOR_SET_COLOR(24, 0x33, 0x66, 0x99);
+            RGB_MATRIX_INDICATOR_SET_COLOR(25, 0x33, 0x66, 0x99);
+            RGB_MATRIX_INDICATOR_SET_COLOR(26, 0x33, 0x66, 0x99);
+        }
+    }
```
Comment (drashna) [inline keyboards/dztech/dz60rgb/keymaps/piv3rt/keymap.c:231]: To match the above
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/dztech/dz60rgb/keymaps/piv3rt/keymap.c (hunk 2 above)
Comment (piv3rt) [reply keyboards/dztech/dz60rgb/keymaps/piv3rt/keymap.c]: Done 👍

File: keyboards/dztech/dz60rgb/keymaps/piv3rt/keymap.c (hunk 3 above)
Comment (piv3rt) [reply keyboards/dztech/dz60rgb/keymaps/piv3rt/keymap.c:229]: Done 👍

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**